TESTS/mbed_hal/ticker/main.cpp

```diff
@@ -1986,6 +1986,34 @@ static void test_irq_handler_insert_non_immediate_in_irq()
     TEST_ASSERT_EQUAL(0, interface_stub.disable_interrupt_call);
 }
 
+static uint32_t ticker_interface_stub_read_interrupt_time()
+{
+    ++interface_stub.read_call;
+    // only if set interrupt call, to test the condition afterwards
+    if (interface_stub.set_interrupt_call) {
+        return interface_stub.interrupt_timestamp;
+    } else {
+        interface_stub.timestamp;
+    }
+} 
+
+static void test_set_interrupt_past_time()
```

Comment:
Could you comment what the test do like it is done for other test. The documentation (and test) is supposed to act as a specification of the code behavior .